voxblox/include/voxblox/integrator/tsdf_integrator.h

```diff
@@ -149,6 +149,18 @@ class SimpleTsdfIntegrator : public TsdfIntegratorBase {
                          const Pointcloud& points_C, const Colors& colors,
                          const bool freespace_points,
                          ThreadSafeIndex* index_getter);
+
+  /*TODO(mereweth@jpl.nasa.gov) - figure out how to bind this in pybind11
+   * as part of the base class
+   */
```

Comment:
nit: we use `//` even for multi line comments. Are these TODOs still valid?

Reply:
If you're ok with it, I'll move those fillSphere and clearSphere functions to Layer, specialized for Layer<EsdfVoxel> the way they are. For TsdfVoxel, I'll think about what makes sense for editing a layer manually in projective distance.